npm/private/npm_translate_lock.bzl

```diff
@@ -766,6 +780,11 @@ load("@aspect_rules_js//npm/private:npm_package_store.bzl", _npm_package_store =
         maybe_bins = ("""
         bins = %s,""" % starlark_codegen_utils.to_dict_attr(_import.bins, 2)) if len(_import.bins) > 0 else ""
 
+        if _import.package.startswith("@"):
+            scope = _import.package.split("/")[0]
```

Comment:
nit: you can specify maxsplit as the 2nd param
```
scope = _import.package.split("/", 1)[0]
```
if you know ahead of time you only ever want to split once